### PR TITLE
Updates to d3

### PR DESCRIPTION
**Pull request by mtraynham**
Fixes Nest's map function & DSV's parse function.

### CODE REVIEW

File: d3/d3.d.ts [hunk 1]
```diff
@@ -702,8 +702,9 @@ declare module D3 {
         * Parse a delimited string into objects using the header row.
         *
         * @param string delimited formatted string to parse
+        * @param accessor to modify properties of each row
         */
-        parse(string: string): any[];
+        parse(string: string, accessor?: (row: any) => any): any[];
```
Comment (vvakame) [inline d3/d3.d.ts]: https://github.com/mbostock/d3/blob/1398d4d8a3517e8a9d33939abad6b1a4706d11e3/src/dsv/dsv.js#L31
accessor has second argument.

Comment (mtraynham) [inline d3/d3.d.ts]: Good catch, looks like it's the index.  I'll make it optional.
```
parse(string: string, accessor?: (row: any, index?: number) => any): any[];
```

Comment (vvakame) [inline d3/d3.d.ts]: :+1:

**Comment by vvakame** [comment]
---
note
https://github.com/mbostock/d3/blob/48ad44fdeef32b518c6271bb99a9aed376c1a1d6/src/arrays/nest.js#L65

**Comment by mtraynham** [comment]
@vvakame Everything good with the merge?

**Comment by vvakame** [comment]
@mtraynham thanks mate! sorry i was late..

**Comment by mtraynham** [comment]
@vvakame No problem, just wondered if you had some concerns.  Thanks again!

**Pull request merged**